test/epoll1.c

```diff
@@ -118,15 +118,26 @@ main(int argc, char *argv[])
   // sockaddr.sin_addr.s_addr = INADDR_LOOPBACK;
   sockaddr.sin_port = htons(port);
 
-  listen_sock = socket(AF_INET, SOCK_STREAM, 0);
+  listen_sockfd = socket(AF_INET, SOCK_STREAM, 0);
   do {
     sockaddr.sin_port = htons(ntohs(sockaddr.sin_port) + 1);
-    rc = bind(listen_sock, (struct sockaddr *)&sockaddr, sizeof(sockaddr));
+    rc = bind(listen_sockfd, (struct sockaddr *)&sockaddr, sizeof(sockaddr));
   } while (rc == -1 && errno == EADDRINUSE);
   if (rc == -1) {
     perror("bind");
+    exit(1);
   }
-  listen(listen_sock, 5);
+  listen(listen_sockfd, 5);
+
+  // This is informmational only; it could be deleted.
```

Comment:
informmational -> informational

Reply:
The code was failiing due to a bug in the first line of 'main()'.  The test epoll2 in autotest.py says to run `./test/epoll1 --use-epoll-create`.  The ffirst line of main() says `int port = (argc == 2 ? atoi(argv[1]) : 6500);`  `argv[1]` is a `char *`.  But atoi converts that to an `int` without any type checking.  So, `atoi()` is wrong when it is give `"--use-epoll-create"` as its argument.

Lower down, the code starts with port 6500, and then successively looks at 6501, 6502, ..., until it finds a ffree port.  That part of the code seems correct.

Originally, whoever wrote epoll1.c was probably intending that it would take at most one argument:  a port number.  Later, they forgot that, and decided that it would take at most one argument, the string `"--use-epoll-create"`.

The rest of my changes were just small cleanups (e.g., sockfd instead of sock, to correspond to the man page).  I'll update `informmational`.  Thanks for that fix.

Reply:
Thanks for the clarification, Gene.

LGTM, after the typo fix.